### ISSUE TITLE
2 sidebar blocks stacked affects the padding

### ISSUE THREAD

**Issue by wturnbull** [CuBoulder/tiamat-theme#1038]
On the grad degree pages, there are 2 blocks stacked in the sidebar. The second block in light blue has no top padding. I don't see this issue when there is only one block in the sidebar. This is a new issue, after latest sandbox update.

Grad page: https://live-ucbdev-homepage.pantheonsite.io/academics/grad-accounting-taxation
![Screenshot 2024-06-06 at 9 55 34 AM](https://github.com/CuBoulder/tiamat-theme/assets/1757082/4d4a2db4-e8fc-4e52-9375-f56598e69b71)

Undergrad page with just one block:
![Screenshot 2024-06-06 at 9 57 42 AM](https://github.com/CuBoulder/tiamat-theme/assets/1757082/5f0e9b8b-4ca8-44d0-b7cf-f0bd2fd349d2)

**Comment by timurtripp** [comment]
@jnicholCU May be related to the padding issues I found. Are we still waiting to hear back from the UX team on this?

**Comment by patrickbrown-io** [comment]
This was specifically requested functionality to allow for more complex block stacking: https://github.com/CuBoulder/tiamat-theme/issues/820

**Comment by jnicholCU** [comment]
I'm still waiting on info from kevin about spacing for top and bottom margin/padding.
He's out this week so I'll know early next week. I'll go ahead and assign myself to this one so no one else gets confused.
